packages/manager/modules/nasha/src/dashboard/partition/accesses/accesses.routing.js

```diff
@@ -0,0 +1,25 @@
+import { createTaskTrackerStateOptions } from '../../../components/task-tracker';
+import { STATE_NAME } from './accesses.constants';
```

Comment:
As this is routing and ui router related, imo it is better just to create a 
```
const stateName = xxx
```
in this file if it's to avoid duplication instead of mixing it with plain constants